components/search/MoreFacetsDropdownOptions.vue

```diff
@@ -67,19 +68,41 @@
       name: {
         type: String,
         required: true
+      },
+      preSelected: {
+        type: Array,
+        default: () => []
       }
     },
 
     data() {
       return {
+        selected: [],
         isActive: false,
         limitTo: 9
       };
     },
 
+    watch: {
+      preSelected(value) {
+        this.selected = value;
+      }
+    },
+
+    mounted() {
+      this.$root.$on('updateSelectedOptions', this.updateOptions);
```

Comment:
This is not the way to have parent components communicate data changes to children. Refactor to rely on the props changing.